### PR TITLE
Genprint v0.4

### PR DESCRIPTION
**Pull request by progman1**
library for printing of values vis a vis the ocaml interpreter, but from within compilation units.

### CODE REVIEW

File: packages/genprint/genprint.0.3/opam [hunk 1]
```diff
@@ -26,6 +26,7 @@ depends: [
 ]
 
 url {
-  src: "https://github.com/progman1/genprintlib/archive/v0.3a.tar.gz"
-  checksum: "md5=d5b58cc0aa7e1dd745d94fcc405d9d27"
+  src: "https://github.com/progman1/genprintlib/archive/v0.3.tar.gz"
+
+  checksum: "md5=0f577477a139c27de8acbc9d5a878562"
```
Comment (kit-ty-kate) [inline packages/genprint/genprint.0.3/opam:31]: I'm sorry but we already talked about this in https://github.com/ocaml/opam-repository/pull/16199. Could you please not change previous releases? Your new `genprint.0.4` is fine though, why change the previous one?

Comment (progman1) [reply packages/genprint/genprint.0.3/opam:31]: I updated the checksum but forgot to bump the src line release file number so opam file pointed to wrong tarball.

Comment (progman1) [reply packages/genprint/genprint.0.3/opam:31]: the process is confusing - that was back when I tried to correct the 0.3 release thinking it was not even installable.
If I revert the details, is that safe to do?!

Comment (kit-ty-kate) [reply packages/genprint/genprint.0.3/opam:31]: yes

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**